cmake/CheckFunctionKeywords.cmake

```diff
@@ -1,14 +1,13 @@
 include(CheckCSourceCompiles)
 
 macro(check_function_keywords _wordlist)
-  set(${_result} "")
   foreach(flag ${_wordlist})
     string(REGEX REPLACE "[-+/ ()]" "_" flagname "${flag}")
     string(TOUPPER "${flagname}" flagname)
     set(have_flag "HAVE_${flagname}")
     check_c_source_compiles("${flag} void func(); void func() { } int main() { func(); return 0; }" ${have_flag})
-    if(${have_flag} AND NOT ${_result})
-      set(${_result} "${flag}")
-    endif(${have_flag} AND NOT ${_result})
+    if(${have_flag})
+      break()
```

Comment:
There was no `break()` in original code. Seems that the result of calling this macro and hence the macro itself is not used (?)

Reply:
OK I was confused at first myself. But then I saw how it works. The `_result` variable seems to be unused entirely (probably copied from a CMake macro example and never removed). The `have_flag` variable is used to set one of the following variables: `HAVE_INLINE`, `HAVE__INLINE`, or `HAVE__INLINE__`. Once it defines that variable, it stores the flag in `${_result}`, and since `_result` has the value `""`, we see `${""} = "inline"`.

Meanwhile, the only reason for using `_result` seems to be to guarantee only one of the variables are set. Without result, we might see `HAVE_INLINE` and `HAVE__INLINE__` both defined. Therefore, I added a break after the first `have_flag` variable is defined.